plugins/field-bitmap/src/field-bitmap.js

```diff
@@ -332,7 +352,7 @@ export class FieldBitmap extends Blockly.Field {
 
   /**
    * Constructs an array of zeros with the specified width and height
-   * @return the new value
+   * @return {!Array.<Array<?>>}the new value
```

Comment:
In this array, we know the values are all 0s, so instead of `?` you can use `number` like

`!Array<!Array<number>>`